reform_cage.py

```diff
@@ -0,0 +1,176 @@
+"""
+From a two-component cage .mol file, returns:
+
+	1) The SMILES code of the building blocks.
+	1.1) The active coordination number of the linkers.
+        1.2) The active functional group.
+        1.3) The number of times a building block has been used in a cage.
+
+	2) The topology of the cage.
+
+        3) Optional Arguments.
+        3.1) Optional (-c): Reform the unrelaxed cage from .mol file.
+        3.2) Optional (-b): Reform the unrelaxed building-blocks (bb).
+
+Usage: $ python reform_cage.py <file.mol>
```

Comment:
This is not the correct usage message because it does not contain all the optional paramters and stuff you defined through argparse. Despite the Python PEP sayiing the docstring should contain the usage string, I would say maybe do 
something like Usage: python reform_cage.py --help
because if the user writes that they will see the full usage message and you dont have to keep it up to date.